xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -971,115 +1000,10 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-    <Member MemberName="System.Collections.ICollection.CopyTo">
```

Comment:
I *think* it was correct to remove this one because there are already 2 implementations documented in this file (lines [112 ](https://github.com/dotnet/dotnet-api-docs/pull/2803/files#diff-69e2ff9cb3ca19ddcb83db53484b00ceR112) and [186](https://github.com/dotnet/dotnet-api-docs/pull/2803/files#diff-69e2ff9cb3ca19ddcb83db53484b00ceR186)), one for an `Array` as first parameter and another one for a `Group[]` as first parameter.

This third version may have been mistakenly added by mdoc back in January. Notice the MemberName.

Reply:
Ok thank you for letting me know. I'll restore them.